samples/posix/gettimeofday/prj.conf

```diff
@@ -30,3 +30,7 @@ CONFIG_NET_CONFIG_MY_IPV4_GW="192.0.2.2"
 
 # Network debug config
 CONFIG_NET_LOG=y
+
+# Use realtime for log timestamps
+CONFIG_LOG_TIMESTAMP_USE_REALTIME=y
+CONFIG_POSIX_C_LANG_SUPPORT_R=y
```

Comment:
This is interesting. I'm really hoping we can use the custom profile to make this the default in Zephyr. Consistence with both the coding guidelines and spec would be👌